newIDE/app/src/SceneEditor/index.js

```diff
@@ -546,13 +546,15 @@ export default class SceneEditor extends React.Component<Props, State> {
 
   _onSelectInstances = (
     instances: Array<gdInitialInstance>,
+    multiSelect: boolean,
     centerView: boolean = true
```

Comment:
Seems that this 3rd argument is unused (always true), we might think of removing it.

Reply:
Yes, It wasn't used anywhere. I removed it.